portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me</title>
+     <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+
+  
+
+  </head>
+  <body>
+     
+      <h1 class="jumbotron"> <center> About Me</center> </h1>
```

Comment:
The indentation starting from this line is a little off for this file and others. Please see https://google.github.io/styleguide/htmlcssguide.html#Indentation

Each nested element should be indented only once.